### PR TITLE
Add login

### PR DESCRIPTION
**Pull request by krzjez**

### CODE REVIEW

**Review by KR4K0 — APPROVED** [review]

File: src/main/java/com/metabug/web/dto/LoginDto.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    @Size(min = 1)
+    private String login;
+
+    @NotNull
+    @Size(min = 1)
+    private String password;
+
+    @Override
+    public String toString() {
+        return "UserDto{" +
```
Comment (kujawad) [inline src/main/java/com/metabug/web/dto/LoginDto.java]: LoginDto

File: src/main/java/com/metabug/web/controller/IndexController.java [hunk 2]
```diff
@@ -16,9 +34,35 @@ public ModelAndView index() {
...
+        if(bindingResult.hasErrors()) {
+            addWarning(modelAndView);
+            modelAndView.setViewName("index");
+            return modelAndView;
+        }
+
+        final User user = userService.findUserByLogin(loginDto.getLogin());
+
+        if(user != null) {
+            if(bCryptPasswordEncoder.matches(loginDto.getPassword(), user.getPassword())) {
```
Comment (kujawad) [inline src/main/java/com/metabug/web/controller/IndexController.java]: Maybe remove field `bCryptPasswordEncoder` from class and add method in `UserService` which will compare passwords using it's `bCryptPasswordEncoder` field?

**Review by kujawad — CHANGES_REQUESTED** [review]

File: src/main/java/com/metabug/web/dto/LoginDto.java (hunk 1 above)
Comment (krzjez) [reply src/main/java/com/metabug/web/dto/LoginDto.java]: done

File: src/main/java/com/metabug/web/controller/IndexController.java (hunk 2 above)
Comment (krzjez) [reply src/main/java/com/metabug/web/controller/IndexController.java]: done

**Review by kujawad — APPROVED** [review]

**Pull request merged**